src/main/java/com/maxtrain/ersjava/blueteam/expense/ExpenseController.java

```diff
@@ -67,27 +67,7 @@ public ResponseEntity putExpense(@PathVariable int id, @RequestBody Expense expe
 		return new ResponseEntity<>(HttpStatus.NO_CONTENT);
 	}
 
-
```

Comment:
Removed PayExpense V 1

Reply:
This is intentional

Reply:
Is version 2 better?

Reply:
Yes. Only change is the addition of logic to ensure only "APPROVED" Expenses get paid.